app/src/control_msg.c

```diff
@@ -180,7 +183,7 @@ sc_control_msg_log(const struct sc_control_msg *msg) {
             } else {
                 // numeric pointer id
                 LOG_CMSG("touch [id=%" PRIu64_ "] %-4s position=%" PRIi32 ",%"
-                             PRIi32 " pressure=%g buttons=%06lx",
+                             PRIi32 " pressure=%f buttons=%06lx",
```

Comment:
`%g` doesn't work on Windows (maybe a MinGW issue), so changed to `%f`.

Reply:
Weird, it should work :/

I keep `%g` for now and it will need to be solved separately. I will test later.

Reply:
I confirm that `%g` does not work with MinGW, I can reproduce.

https://github.com/Genymobile/scrcpy/pull/3399